addons/crm/views/crm_lead_views.xml

```diff
@@ -633,10 +633,10 @@
                     <div class="o_kanban_record_subtitle">
                         <t t-if="record.prorated_revenue.raw_value">
                             <field name="prorated_revenue" widget="monetary" options="{'currency_field': 'company_currency'}"/>
-                            <span t-if="record.recurring_revenue and record.recurring_revenue.raw_value"> + </span>
+                            <span t-if="record.recurring_revenue_prorated and record.recurring_revenue_prorated.raw_value"> + </span>
                         </t>
-                        <t t-if="record.recurring_revenue and record.recurring_revenue.raw_value">
-                            <field name="recurring_revenue" widget="monetary" options="{'currency_field': 'company_currency'}"/>
+                        <t t-if="record.recurring_revenue_prorated and record.recurring_revenue_prorated.raw_value">
```

Comment:
Note how the t-if construct is not optimal, as we are testing twice for the same thing. This could be used by inheriting templates, so we shouldn't look into this now, but it could be worth exploring it for master. (In which case we should remind ourselves to stop the forward port before master to improve on it if it's still applicable).